### DISCUSSION TITLE
WebView2 is not working on NET 5.0

### DISCUSSION THREAD

**Discussion by wgtgithub** [MicrosoftEdge/WebView2Feedback#3924] [Q&A] [answered]
I created the following WPF projects
1. User Control
2.  Windows App

And I deployed WebView2 on a  User control and deployed it on the windows app.
I tried to run the app...the app show web browser but soon the screen became white....
I tried it on .Netframework 4.7 also and it was good working....

**Comment by champnic** [comment]
Hey @wgtgithub - do you have an example app which repros this issue?

When you say "screen became white", do you mean it was working fine and then all of sudden turned white? Was it still interactable, but the content was invisible?

My hunch is that this might be a graphics driver issue. Thanks!

**Comment by wgtgithub** [comment, marked as answer]
Thanks, your replaying
I fixed this matter already
Actually, the reason was usage of event handling between JavaScript and Webview2 action 
So my webVeiew2 is good working now
